.env.default

```
@@ -0,0 +1,7 @@
+SMTP_USERNAME=
+SMTP_PASSWORD=
+MAIL_TESTER_FROM=
+MAIL_TESTER_TO=
+MAIL_TESTER_BCC=
+MAIL_TEST_SUBJECT=
+MAIL_TEST_MESSAGE=
```

Add New line end of file.

```suggestion
MAIL_TEST_MESSAGE=

```